java-grupa8/homework-2/src/sort.java

```diff
@@ -0,0 +1,35 @@
+/**
+ Implements a bubble sort algorithm to sort an array of SalesRepresentative objects in descending order.
+ The sorting should be done based on the amount of revenue generated via sales.
+ Each rep has a number of sales, and a quota / sale.
+ */
+public class sort {
```

Comment:
All classes must start with capital letters. Please check the naming conventions.